java/client/src/org/openqa/selenium/remote/RemoteWebElement.java

```diff
@@ -349,13 +349,16 @@ public Dimension getSize() {
     return new Dimension(width, height);
   }
 
+  @SuppressWarnings({"unchecked"})
   public Rectangle getRect() {
     Response response = execute(DriverCommand.GET_ELEMENT_RECT, ImmutableMap.of("id", id));
     Map<String, Object> rawRect = (Map<String, Object>) response.getValue();
-    int x = ((Number) rawRect.get("x")).intValue();
-    int y = ((Number) rawRect.get("y")).intValue();
-    int width = ((Number) rawRect.get("width")).intValue();
-    int height = ((Number) rawRect.get("height")).intValue();
+    Map<String, Object> origin = (Map<String, Object>) rawRect.get("origin");
+    Map<String, Object> size = (Map<String, Object>) rawRect.get("size");
```

Comment:
The return changed to sth like:
`{"value":{"origin":{"x":108,"y":248},"size":{"width":552,"height":21}},"sessionId":"eb26b690-8329-4d24-872e-0610b7c69a76","status":0}`